packages/contracts/contracts/core/permission/PermissionManager.sol

```diff
@@ -343,7 +343,10 @@ contract PermissionManager is Initializable {
     /// @notice A modifier to be used to check permissions on a target contract.
     /// @param _where The address of the target contract for which the permission is required.
     /// @param _permissionId The permission identifier required to call the method this modifier is applied to.
-    function _auth(address _where, bytes32 _permissionId) private {
+    function _auth(address _where, bytes32 _permissionId) private view {
+        // TODO: will we need to emit `who` as well ? below `who: msg.sender` 
+        // is not actul who that we grant the permission in the end, it's the sender
+        // that initiations permission.
```

Comment:
Let's not modify the permission manager in this PR and do it in a separate PR, e.g., https://github.com/aragon/core/pull/90.
Please put the comments there.

Reply:
Noted. Though, I will leave `'view` keyword.  Not a big overhead, comment removed + test that was brought for our new bug to fix it removed as well (it's in another PR).

Reply:
Ok